src/pkg/clouds/do/appPlatform/setup.go

```diff
@@ -92,8 +102,12 @@ func shellQuote(args ...string) string {
 	return strings.Join(quoted, " ")
 }
 
+func getCdImagePath(imageTag string) string {
```

Comment:
Let's make sure the AWS and DO funcs have the exact same name. We could probably do away with the 2 references to `pkg.Getenv("DEFANG_CD_IMAGE",…)` by adding a function to byoc package